augur/export_v2.py

```diff
@@ -874,7 +902,15 @@ def parse_node_data_and_metadata(T, node_data_files, metadata_file):
                 node_attrs[name][corrected_key] = value
                 node_data_names.add(corrected_key)
 
-    return (node_data, node_attrs, node_data_names, metadata_names)
+    # third pass: create `branch_attrs` which includes certain traits supplied in `node_attrs`
+    # (e.g. mutations are coverted to branch attrs, and `clade_annotation` is interpreted as a label)
```

Comment:
Should this "third pass" reference be removed? It seems like a copy/paste from earlier in the module.

This comment also refers to `clade_annotation`, but that won't exist anymore, right?

Reply:
Done 👍